### ISSUE TITLE
Training on ScanNet

### ISSUE THREAD

**Issue by sophiajw** [erikwijmans/Pointnet2_PyTorch#56]
Hi all,

has anybody tried to apply the network on ScanNet data?
I am currently trying to achieve similar performance as mentioned in PointNet++ paper, which is around 83% accuracy. However, when training with this model, I can hardly achieve values greater than 60%.

I tried using the same hyperparameters as in the original PointNet++ setting (i.e. changed lr to 1e-3). 

Any suggestions or help would be appreciated.

Best,
Sophia

**Comment by erikwijmans** [comment]
If I recall correctly, I was able to reproduce the results on ScanNet by setting the hyper-parameters to the same as in https://github.com/charlesq34/pointnet2/blob/master/scannet/train.py **and** adding the class balanced cross-entropy loss used there.

**Comment by daveredrum** [comment]
Hi there,

I'm facing the same problem of reproducing the performance in the PointNet++ paper. The best accuracy I had is 59%.

I use the code in [https://github.com/charlesq34/pointnet2/blob/master/scannet/preprocessing/collect_scannet_scenes.py](https://github.com/charlesq34/pointnet2/blob/master/scannet/preprocessing/collect_scannet_scenes.py) to preprocess the ScanNet data and the data loader in [https://github.com/charlesq34/pointnet2/blob/master/scannet/scannet_dataset.py](https://github.com/charlesq34/pointnet2/blob/master/scannet/scannet_dataset.py) to fetch the data.

I'm wondering if anyone could share the code for training on ScanNet with this PointNet++ implementation? That'd be super helpful. :)

Best,
Dave
